specification/eventgrid/resource-manager/Microsoft.EventGrid/preview/2020-10-15-preview/EventGrid.json

```diff
@@ -0,0 +1,8286 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2020-10-15-preview",
+    "title": "EventGridManagementClient",
+    "description": "Azure EventGrid Management Client"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.EventGrid/domains/{domainName}": {
+      "get": {
+        "tags": [
+          "Domains"
+        ],
+        "summary": "Get a domain.",
+        "description": "Get properties of a domain.",
+        "operationId": "Domains_Get",
+        "consumes": [],
```

Comment:
I saw the global definition of "consumes" and 'produces',  do you still need to add it into each operation?

Reply:
It is not clear to us what is the ask here. Please note that this is auto-generated code/spec for us and this is existing resource/section from previous swagger (please refer to https://github.com/Azure/azure-rest-api-specs/blob/master/specification/eventgrid/resource-manager/Microsoft.EventGrid/preview/2020-04-01-preview/EventGrid.json#L12 )... 

This is outside scope of this PR. We will not fix it as part of this PR as it is specific for new features/resources. If any fix, it should be through the provided tools for services team. thanks